neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -9,7 +9,7 @@ use std::{cell::RefCell, rc::Rc};
 use neqo_common::Encoder;
 use neqo_transport::{recv_stream, send_stream, Connection, StreamId};
 
-use super::WebTransportSession;
+use super::webtransport_session;
```

Comment:
```suggestion
use super::webtransport::session;
```
I'd like it looking like this, but it may mean too much refactoring :-)